src/main/resources/application.yaml

```diff
@@ -75,3 +75,6 @@ azure:
   blob:
     connection-string: ${CONNECTION_STRING}
     container-name: files
+
+cron:
+  media-application-reporting: "0 12 */30 * *"
```

Comment:
would probably be best to run this at the end of the day as if you run it here youll miss any that come after
`59 23 30 * *` might be best to do here

Reply:
Done